tests/test_client.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+from  dotenv import load_dotenv
+import pytest
+
+import fairly
+
+
+load_dotenv('./.env')
+FIGSHARE_TOKEN = os.environ['FAIRLY_FIGSHARE_TOKEN']
+DUMMY_DATASET = "./tests/dummy_dataset/"
+
+def test_config():
+    """
+    Test the config file
+    """
+    import fairly
+
+    config_dir = os.path.expanduser("~/.fairly/")
+    config_file = os.path.join(config_dir, "config.json")
+
+    # Check that local global configuration is set when fairly is imported
+    assert os.path.exists(config_dir), "~/fairly dir doesnt exist"
+    assert os.path.exists(config_file), "config.json doesnt exist"
+
+    # delete ~/.fairly/ directory and its contents
+    for file_name in os.listdir(config_dir):
+        file_path = os.path.join(config_dir, file_name)
+        if os.path.isfile(file_path):
+            os.remove(file_path)
```

Comment:
This is a very risky operation, as it deletes a file from the home folder of the user. If the user is actively using fairly (i.e. has a valid custom configuration file), it deletes the actual configuration.